images.yaml

```
@@ -76,6 +76,9 @@
   - sha: 9029b2f52ecd28aacec2fd8a86a321dc9f77a46df251de5e3f157dd6e80baea0
     tag: 0.12
 - name: coredns/coredns
+  tags:
+  - sha: db4f1c57978d7372b50f416d1058beb60cebff9a0d5b8bee02bfe70302e1cb2f
```

```suggestion
  - sha: 642ff9910da6ea9a8624b0234eef52af9ca75ecbec474c5507cb096bdfbae4e5
```
Confusingly, the `DIGEST` shown on Docker Hub isn't actually the digest to use to identify the image. We need to use the `RepoDigest` which is shown when you pull the image, or `docker image inspect` it.
e.g.
```
$ docker pull coredns/coredns:1.8.3
1.8.3: Pulling from coredns/coredns
c6568d217a00: Pull complete 
a9b089d7de15: Pull complete 
Digest: sha256:642ff9910da6ea9a8624b0234eef52af9ca75ecbec474c5507cb096bdfbae4e5
Status: Downloaded newer image for coredns/coredns:1.8.3
docker.io/coredns/coredns:1.8.3
```

or

```
$ docker image inspect coredns/coredns:1.8.3
[
    {
        "Id": "sha256:3885a5b7f138c3e8779a18f672fd55c3c7cf0166b67cf5143ab0cd137eb38418",
        "RepoTags": [
            "coredns/coredns:1.8.3"
        ],
        "RepoDigests": [
            "coredns/coredns@sha256:642ff9910da6ea9a8624b0234eef52af9ca75ecbec474c5507cb096bdfbae4e5"
        ],
        ...
     }
]
```

---

Thanks!